index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class="header-bg">
+            <img class="Ellipse12" src="images/Ellipse 12.png" alt="">
+            <img class="Ellipse13" src="images/Ellipse 13.png" alt="">
+            <img class="Ellipse4" src="images/Ellipse 4.png" alt="">
+            <img class="Rectangle5" src="images/Rectangle 5.png" alt="">
+            <img class="Ellipse3" src="images/Ellipse 3.png" alt="">
+            <img class="Rectangle4" src="images/Rectangle 4.png" alt="">
+            <img class="Rectangle7" src="images/Rectangle 7.png" alt="">
+            <img class="Ellipse5" src="images/Ellipse 5.png" alt="">
+            <img class="Ellipse11" src="images/Ellipse 11.png" alt="">
+            <img class="Rectangle6" src="images/Rectangle 6.png" alt="">
+        </div>
+        <section class="header">
+            <label class="logo" >My Logo</label>
+            <ul>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </section>
+
+        <section class="main">
+            <div class="Text">
+                <div class="header2">
+                    <h1>I'm Alex Waweru. Glad to see you!</h1>
+                </div>
+                <div class="primary-text1">
+                    <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                        don't hesitate to contact me.
+                       </p>
```

Comment:
- [ ] Kindly ensure that your paragraph text has the right `font-size`, margins, and paddings (top, bottom, left, right) defined in the design. :100: 
- [ ] Please ensure that your design looks similar to that of the Figma design as specified in the Project Requirements as I mentioned before :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/227491816-87df1936-43e8-4f5d-8b8e-fcb6919ec7b5.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/227490077-2750aa09-2e11-4f57-8ece-4979d5679a02.png)